packages/ipa-certified-attributes-importer/.env

```diff
@@ -11,6 +11,7 @@ AOO_URL="https://indicepa.gov.it/ipa-dati/datastore/dump/cdaded04-f84e-4193-a720
 UO_URL="https://indicepa.gov.it/ipa-dati/datastore/dump/b0aa1f6c-f135-4c8a-b416-396fed4e1a5d?format=json"
 INSTITUTIONS_CATEGORIES_URL="https://indicepa.gov.it/ipa-dati/datastore/dump/84ebb2e7-0e61-427b-a1dd-ab8bb2a84f07?format=json"
 
+ECONOMIC_ACCOUNT_COMPANIES_ALLOWLIST=05GYSWFH
```

Comment:
values in `.env` should not be used in tests (unless under the `test` folder) to avoid breaking the tests.
I suggest to pass the configuration object as parameter in order to fix values required in tests